Highlight migration doesn't work with one KRaft controller ephemeral storage (#10190)
strimzi/strimzi-kafka-operator: documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

* You must be using Strimzi 0.40 or newer with Kafka 3.7.0 or newer. If you are using an earlier version of Strimzi or Apache Kafka, upgrade before migrating to KRaft mode.
* Verify that the ZooKeeper-based deployment is operating without the following, as they are not supported in KRaft mode:
** JBOD storage. While the `jbod` storage type can be used, the JBOD array must contain only one disk.

> * The current running brokers and the controllers to be deployed are not configured with `ephemeral` storage which is not supported for migration purposes.

Reviewer: ```suggestion
* The running brokers and the controllers to be deployed must not be configured with `ephemeral` storage, which is not supported for migration.
```

To migrate when the cluster is not using node pools, brokers must be contained in a `KafkaNodePool` resource configuration that is assigned a `broker` role and has the name `kafka`.
Support for node pools is enabled in the `Kafka` resource configuration using the `strimzi.io/node-pools: enabled` annotation.

> IMPORTANT: Migrating to KRaft by using just one controller with `ephemeral` storage isn't going to work.
> On controller restart, during the migration process, it's going to lose the metadata synced from ZooKeeper (that is, topics, ACLs, and so on).
> In general, migrating an ephemeral based ZooKeeper cluster to KRaft isn't recommended.

Reviewer: ```suggestion
IMPORTANT: Using a single controller with ephemeral storage for migrating to KRaft will not work. 
During the migration, controller restart will cause loss of metadata synced from ZooKeeper (such as topics and ACLs). 
In general, migrating an ephemeral-based ZooKeeper cluster to KRaft is not recommended.
```
